TwitchPlaysAssembly/Src/Commands/IRCConnectionManagerCommands.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections;
+using UnityEngine;
+
+namespace TwitchPlaysAssembly.Src.Commands
+{
+	/// <summary>Commands for the IRC Connection Holdable.</summary>
+	public static class IRCConnectionManagerCommands
+	{
+		[Command(@"help")]
+		public static IEnumerator Help()
+		{
+			var helpMessage = "Disconnect the IRC from twitch plays with “!{0} disconnect”. For obvious reasons, only the streamer may do this.";
+			//if (TPElevatorSwitch.Instance != null && TPElevatorSwitch.Instance.gameObject.activeInHierarchy)
+			//	helpMessage += " Turn the elevator on with “!{0} elevator on”. Turn the elevator off with “!{0} elevator off”. Flip the elevator on/off with “!{0} elevator toggle”.";
```

Comment:
Because I removed the `!ircmanager elevator on/off` command. Nobody was using it anyway. Only `!elevator on/off` remains, and is separate from the IRC Connection Manager.

Reply:
Should we remove the elevator switch entirely then?